rust/sleeper_core/src/datafusion/region.rs

```diff
@@ -15,7 +15,7 @@
 * limitations under the License.
 */
 use crate::{ColRange, PartitionBound};
-use datafusion::logical_expr::{Expr, col, lit};
+use datafusion::logical_expr::{Expr, ident, lit};
```

Comment:
It looks like this change stops it normalising the column names? It looks like that means it was converting them to lower case before. Was this a bug? Where is it tested?